testing/trino-testing/src/main/java/io/trino/testing/containers/HiveHadoop.java

```diff
@@ -0,0 +1,107 @@
+/*
```

Comment:
OK moved to trino-hive

Reply:
Not feeling strongly - but I would not say `HiveHadoop` is specific to hive-connector. There are other connectors which use hive metastore as a metadata storage, or HDFS as data storage. E.g Iceberg.

Reply:
@losipiuk had same feeling. So I've checked if this can be done. And as e.g iceberg has trino-hive as dependency so this class will still be available.